### ISSUE TITLE
Partial Attribute Update does not update modifiedAt value

### ISSUE THREAD

**Issue by idomingu** [ScorpioBroker/ScorpioBroker#188]
Hi everyone,

While testing ```scorpio-aaio_1.0.3``` Docker image I have noticed that issuing a Partial Attribute Update does not change the value of the associated ```modfiedAt```.

Thanks in advance,

Cheers,
Nacho.

**Comment by amitrnec** [comment]
Kindly share the payload and post/get api request.

**Comment by idomingu** [comment]
Apologies. Here is an example:

1. Create example "Vehicle" entity.

```bash
curl --location --request POST 'http://localhost:9090/ngsi-ld/v1/entities/' \
--header 'Content-Type: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
	"id": "urn:ngsi-ld:Vehicle:A100",
	"type": "Vehicle",
	"brandName": {
		"type": "Property",
		"value": "Mercedes"
	},
	"speed": {
		"type": "Property",
		"value": 80
	}
}'
```
2. Let's retrieve the entity we just created. Request is configured so that system attributes are returned:

```bash
curl --location --request GET 'http://localhost:9090/ngsi-ld/v1/entities/?type=Vehicle&options=sysAttrs' \
--header 'Accept: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"'

```

Which results into the following response:

```json
[
    {
        "id": "urn:ngsi-ld:Vehicle:A100",
        "type": "Vehicle",
        "createdAt": "2021-01-21T16:27:00.742000Z",
        "modifiedAt": "2021-01-21T16:27:00.742000Z",
        "brandName": {
            "type": "Property",
            "createdAt": "2021-01-21T16:27:00.742000Z",
            "value": "Mercedes",
            "modifiedAt": "2021-01-21T16:27:00.742000Z"
        },
        "speed": {
            "type": "Property",
            "createdAt": "2021-01-21T16:27:00.742000Z",
            "value": 80,
            "modifiedAt": "2021-01-21T16:27:00.742000Z"
        }
    }
]
```

3. Issue Partial Attribute Update for the ```speed``` property:
```bash
curl --location --request PATCH 'http://localhost:9090/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A100/attrs/speed' \
--header 'Content-Type: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
    "value": "120",
    "type": "Property"
}'
```

4. Once again, retrieve the entity with system attributes:
```bash
curl --location --request GET 'http://localhost:9090/ngsi-ld/v1/entities/?type=Vehicle&options=sysAttrs' \
--header 'Accept: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"'
```

Which returns the following response:
```json
[
    {
        "id": "urn:ngsi-ld:Vehicle:A100",
        "type": "Vehicle",
        "createdAt": "2021-01-21T16:27:00.742000Z",
        "modifiedAt": "2021-01-21T16:33:59.403000Z",
        "brandName": {
            "type": "Property",
            "createdAt": "2021-01-21T16:27:00.742000Z",
            "value": "Mercedes",
            "modifiedAt": "2021-01-21T16:27:00.742000Z"
        },
        "speed": {
            "type": "Property",
            "createdAt": "2021-01-21T16:27:00.742000Z",
            "value": "120",
            "modifiedAt": "2021-01-21T16:27:00.742000Z"
        }
    }
]
```

Please, notice the ```speed```  value is successfully update but ```modifiedAt``` has not changed. In addition, top ```modifiedAt``` attribute of the entity gets updated to the time of the Partial Attribute Update operation.

**Comment by amitrnec** [comment]
Fixed this issue in #189
